dags/blocksec_plugin/contract_interaction_operator.py

```diff
@@ -62,7 +65,7 @@ def execute(self, context):
         raw_txn = self.function(**self.function_args)\
                              .buildTransaction(dict(
                                nonce=int(self.nonce),
-                               gasPrice = int(max(33,self.web3.eth.gasPrice) *\
+                               gasPrice = int(max(self.max_gas_price, self.web3.eth.gasPrice) *\
```

Comment:
Should this become `min` won't the gas price blowing up always select the larger of the two, which would be `gasPrice` not `max_gas_price` which would be the smaller of the two

Reply:
Yes that correct good catch!